src/main/java/com/getindata/connectors/http/internal/table/lookup/HttpLookupTableSourceFactory.java

```diff
@@ -58,11 +55,6 @@ public DynamicTableSource createDynamicTableSource(Context context) {
             LOOKUP_REQUEST_FORMAT.key()
         );
 
-        EncodingFormat<SerializationSchema<RowData>> encodingFormat =
-            helper.discoverEncodingFormat(SerializationFormatFactory.class,
-                HttpLookupConnectorOptions.LOOKUP_REQUEST_FORMAT
```

Comment:
Shouldn't we remove this constant as well?

Reply:
We do want to keep it. It is used in GenericJsonQueryCreatorFactory :)

We still want to have encodingFormat and LOOKUP_REQUEST_FORMAT but the problem is that it was a "zombie" reference here, a leftover from previous refactoring.